compiler/rustc_borrowck/src/nll.rs

```diff
@@ -135,6 +139,18 @@ pub(crate) fn compute_regions<'a, 'tcx>(
         elements,
     );
 
+    // If requested for `-Zpolonius=next`, convert NLL constraints to localized outlives
+    // constraints.
+    let mut localized_outlives_constraints = LocalizedOutlivesConstraintSet::default();
+    if infcx.tcx.sess.opts.unstable_opts.polonius.is_next_enabled() {
+        polonius::create_localized_constraints(
```

Comment:
Is there any reason that this function can't just return a `LocalizedOutlivesConstraintSet` instead of taking it as a mutable reference?

When polonious=next is *not* enabled, I think it makes sense to have the local var here be `None`, and Some otherwise (but this point is just a nit).

Reply:
No fundamental reason, so I've made this an Option.